metricbeat/module/vsphere/datastore/_meta/fields.yml

```diff
@@ -4,32 +4,66 @@
     datastore
   release: ga
```

Comment:
@ishleenk17 Do we need to remove tag or update the value to `beta`?

Reply:
Not required, the changes will go in GA version